main.py

```diff
@@ -31,27 +31,23 @@ async def on_message(message):
 	global category
 	global userMessage
 
+	# if the message is not a dm or if message is from bot, return (Do not proceed)
+	if (not isinstance(message.channel, discord.channel.DMChannel) or message.author == client.user):
+		return
+	
 	# if its the cancel command reset the bot's state
-	if isinstance(message.channel, discord.channel.DMChannel) and (message.author != client.user) and "&cancel" in message.content:
+	if ("&cancel" in message.content):
 		userInputMode = False
 		category = 0
 		userMessage = ""
 
-	#if the channel is a dm and the author of the message variable is not the bot
-	elif isinstance(message.channel, discord.channel.DMChannel) and (message.author != client.user) and userInputMode != True: 
+	#if the message variable is not the bot
+	elif (userInputMode != True): 
 		msg = await message.channel.send("Hi there! If you need some help, please react to this message so we can get started.\nYou can cancel at any time with &cancel") #sends back the same message (for now, it'll send a helpful response message soon)
 		await msg.add_reaction(tick)
 
-	# if the channel is a dm, the messsage author isn't the bot itself and the message content matches the same str as below (janky but will be moved to a file soon) 
-	elif isinstance(message.channel, discord.channel.DMChannel) and (message.author == 
-		client.user) and (message.content == "-----------------\nWhat category would you place your request under?\n\n"+one+" Moderation\n\n"+two+" Tournaments\n\n"+three+" Mentoring\n\n"+four+" Other\n-----------------") and userInputMode != True:
-		await message.add_reaction(one) #add the reactions to the bot so that the user can select a bot
-		await message.add_reaction(two)
-		await message.add_reaction(three)
-		await message.add_reaction(four)
-
```

Comment:
Removed,
Made the bot add comment ight after the message was sent in on_add_reaction event